hackathon/models.py

```diff
@@ -1,4 +1,6 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
 from accounts.models import CustomUser as User
```

Comment:
As per PEP8 could you please add a blank line between 3rd party and local modules?